packages/components/app/styles/components/avatar.scss

```diff
@@ -0,0 +1,24 @@
+//
+// AVATAR
+//
+// properties within each class are sorted alphabetically
+//
+
+.hds-avatar {
+  align-items: center;
+  box-sizing: border-box;
+  display: inline-flex;
+  height: 32px;
+  justify-content: center;
+  width: 32px;
+}
+
+.hds-avatar img {
+  border-radius: 2px;
+  height: 32px;
+  width: 32px;
```

Comment:
suggestion: Would you want to inherit the height and width values on the img? That way you wouldn't need to repeat them here.

Example:
`height: 32px;` => `height: inherit;`